stan/math/prim/scal/err/check_bounded.hpp

```diff
@@ -28,8 +29,8 @@ namespace stan {
                           const T_high& high) {
           using stan::max_size;
 
-          VectorView<const T_low> low_vec(low);
-          VectorView<const T_high> high_vec(high);
+          scalar_seq_view<const T_low> low_vec(low);
```

Comment:
Don't need these `const` modifiers here because everything's saved as a constant reference and returned as a constant referencey by default.  This'll essentially double specify the return type as `const const T`, which doesn't hurt, but it's simpler without it.